### DISCUSSION TITLE
How to solve this tsconfig error?

### DISCUSSION THREAD

**Discussion by nishu-murmu** [monkeytypegame/monkeytype#3588] [Questions]
![Screenshot 2022-09-23 at 1 41 45 PM](https://user-images.githubusercontent.com/71123337/191918362-b959916a-c1ed-43ec-8883-8955fb32cb41.png)
What is this tsconfig error?
and how to solve this?

**Comment by Miodec** [comment]
Have you tried running the install all script in the bin folder? (`.cmd` for windows and `.sh` for everything else)

**Reply by nishu-murmu** [reply]
@Miodec 
I did that
But whenever I'm changing anything in the code it is showing the below error in macos.
![Screenshot 2022-09-28 at 12 24 25 PM](https://user-images.githubusercontent.com/71123337/192709061-7b3abec1-36f0-4099-acf6-f87278a25193.png)

**Reply by Miodec** [reply]
Have you merged upstream?

**Reply by nishu-murmu** [reply]
In my forked repo it was few commits ahead of the master, is this why that was happening?

**Reply by Miodec** [reply]
Maybe. I had a few commits where this tsconfig error was popping up. Make sure to pull upstream and merge master in, then try again. If that fails ping me on discord so we can do some troubleshooting.

**Reply by nishu-murmu** [reply]
Does this error cause any hindrance in local development?

**Reply by Miodec** [reply]
Well, obviously - your eslint wont work. But this issue is fixed already.

**Reply by nishu-murmu** [reply]
**This was the actual issue.
how to solve the linting issue.**


![Screenshot 2022-09-29 at 5 11 27 PM](https://user-images.githubusercontent.com/71123337/193022456-cc1cb629-58c8-4497-8074-367995cef669.png)
